lib/pfconfig/namespaces/resource/UriFilters.pm

```diff
@@ -1,44 +1,33 @@
-package pfconfig::namespaces::resource::Profile_Filters;
-
+package pfconfig::namespaces::resource::UriFilters;
 =head1 NAME
 
-pfconfig::namespaces::resource::Profile_Filters
+pfconfig::namespaces::resource::UriFilters
```

Comment:
That is exactly what I was about to say...
URI, just as PKI, are acronym / abreviation... they need to be CAPITALIZED..
For the underscore, I think it helps readability...